src/content/changes/highlights/2024-07-21.md

```diff
@@ -0,0 +1,186 @@
+app/compare/v0.2.0...v0.3.0)
+        - Push `kube-downscaler` app to all collections.
+    - [0.2.0](https://github.com/giantswarm/kube-downscaler-
+app/compare/v0.1.0...v0.2.0)
+        - Added `enabled` field in values to disable whole chart if needed.
+
+
+- [dex-app](https://github.com/giantswarm/dex-app)
+    - [1.42.11](https://github.com/giantswarm/dex-app/compare/v1.42.10...v1.42.11)
+        - Default `ingress.tls.clusterIssuer` values to `letsencrypt-giantswarm`
+        - Update `cert-manager.io/cluster-issuer` annotation to use default.
+- [teleport-kube-agent-app](https://github.com/giantswarm/teleport-kube-agent-app)
+    - [0.9.2](https://github.com/giantswarm/teleport-kube-agent-
+app/compare/v0.9.1...v0.9.2)
+        - Introduced `podAntiAffinity` so `teleport-kube-agent` pods run on different `control-plane` nodes also increased the number of replicas to 3 to maintain better high availability.
+
+- [k8s-dns-node-cache-app](https://github.com/giantswarm/k8s-dns-node-cache-app)
+    - [2.8.1](https://github.com/giantswarm/k8s-dns-node-cache-
+app/compare/v2.8.0...v2.8.1)
+        - Make the app visible for all providers.
+
+- [kyverno-policies-connectivity](https://github.com/giantswarm/kyverno-policies-connectivity)
+    - [0.6.0](https://github.com/giantswarm/kyverno-policies-
+connectivity/compare/v0.5.4...v0.6.0)
+        - Update kubectl container image to version v1.26.0 for WorkloadCluster Ip Job
+        - Increase pod and container SecurityContext settings for WorkloadCluster Ip Job
+        - Execute `kubectl apply` with `--server-side=true --field-manager='kubectl-client-side-apply' --force-conflicts` flags in WorkloadCluster Ip Job
+        - Remove unused tests under helm directory.
+- [security-bundle](https://github.com/giantswarm/security-bundle)
+    - [1.8.0](https://github.com/giantswarm/security-bundle/compare/v1.7.1...v1.8.0)
+        - Add `kyverno-crds` app to handle Kyverno CRD install.
+        - Update `kyverno` (app) to v0.17.15. This version disables the CRD install job in favor of `kyverno-crds` App.
+- [kyverno-app](https://github.com/giantswarm/kyverno-app)
+    - [0.17.15](https://github.com/giantswarm/kyverno-
+app/compare/v0.17.14...v0.17.15)
+        - Set VPA max 6 CPU / 24Gi memory and adjust default requests/limits for `reports-controller`.
+        - Set VPA max 4 CPU / 8Gi memory and adjust default requests/limits for `background-controller`.
+        - Set starting CPU limit of request+25% for `cleanup-controller`.
+        - Disable Kyverno CRDs install Job in favor of `kyverno-crds` App.
+- [kyverno-crds](https://github.com/giantswarm/kyverno-crds)
+      - [1.11.1](https://github.com/giantswarm/kyverno-crds/compare/v1.11.0...v1.11.1)
+        - Removed unpopulated labels and fixed the team label.
+      - [1.11.0](https://github.com/giantswarm/kyverno-crds/releases/tag/v1.11.0)
+          - First release of the Kyverno CRDs App for Kyverno v1.11.
```

Comment:
@stone-z  These are from Shield. Please adapt the content to the standard format (name, version, description)
[example](https://raw.githubusercontent.com/giantswarm/docs/main/src/content/changes/highlights/2024-02-15.md)